src/references/api.md

```diff
@@ -0,0 +1,19 @@
+# API reference
+
+Most of the Epinio commands communicate with the Epinio server side component over an API. The API is secured with Basic Auth authentication. You can make requests against that API with the credentials from your Epinio config.
+E.g.
+
+- Get your credentials with: `epinio config show`
+- Use curl to make a request:
+
+```
+$ curl -u your-username-here:password-here https://epinio.your-system-domain/api/v1/info
+```
+
+The responses are all JSON objects, except for some cases when you get an error before the request reaches the Epinio server (e.g. because of bad cluster configuration).
```

Comment:
```suggestion
The responses are all JSON objects, except for some cases where you get an error before the request reaches the Epinio server (e.g. because of bad cluster configuration).
```